enos/modules/vault_cluster/variables.tf

```diff
@@ -1,6 +1,12 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: BUSL-1.1
 
+variable "arch" {
```

Comment:
I don't see where we use the `arch`, `consul_bind_addr`, `distro` or `package_manager` variables in the module.